src/promptflow/promptflow/_sdk/_serving/app.py

```diff
@@ -53,6 +50,14 @@ def init(self, **kwargs):
             logger.info(f"Project path: {self.project_path}")
             self.flow_entity = load_flow(self.project_path)
             self.flow = self.flow_entity._init_executable()
+
+            # enable environment_variables
+            self.environment_variables = kwargs.get("environment_variables", {})
```

Comment:
will this self.environment_variables be used somewhere? if not, maybe we should just use it as a variable here.